serverless/experimental/gcp-log/gcs_uploader/go.sum

```diff
@@ -0,0 +1,595 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I wonder if a `go mod tidy` would help cut the number of versions in here down?

Reply:
Done.